src/main/java/org/maxgamer/quickshop/QuickShop.java

```diff
@@ -1868,7 +1868,10 @@ private void updateConfig(int selectedVersion) throws IOException {
             getConfig().set("shop.use-global-virtual-item-queue", null);
             getConfig().set("config-version", ++selectedVersion);
         }
-
+        if(selectedVersion == 137) {
+            getConfig().set("integration.griefprevention.delete-on-subclaim-created", true);
+            getConfig().set("config-version", ++selectedVersion);
```

Comment:
If you increase the version number, you should also increase the version number in config.yml (138)

Reply:
The version number was increased to 138 in config.yml. Fixed.